crates/l3/appchain-utils/src/lib.rs

```diff
@@ -0,0 +1,110 @@
+pub mod errors;
+use color_eyre::{eyre::eyre, Result};
+use starknet_accounts::{Account, Call, Execution, SingleOwnerAccount};
+use starknet_contract::ContractFactory;
+use starknet_core::types::contract::{CompiledClass, SierraClass};
+use starknet_core::types::{BlockId, BlockTag, FunctionCall, InvokeTransactionResult};
+use starknet_core::utils::get_selector_from_name;
+use starknet_ff::FieldElement;
+use starknet_providers::jsonrpc::{HttpTransport, JsonRpcClient};
+use starknet_providers::Provider;
+use starknet_signers::LocalWallet;
+
+pub type LocalWalletSignerMiddleware =
+    SingleOwnerAccount<JsonRpcClient<HttpTransport>, LocalWallet>;
+
+type RpcAccount<'a> = SingleOwnerAccount<&'a JsonRpcClient<HttpTransport>, LocalWallet>;
+pub type TransactionExecution<'a> = Execution<'a, RpcAccount<'a>>;
+
+pub const NO_CONSTRUCTOR_ARG: Vec<FieldElement> = Vec::new();
+
+pub const MAX_FEE: &str = "0x1000000000000";
```

Comment:
try to create a const of type FieldElement directly, using `FieldElement::from_mont`.
Get the Montgomery repr by printing `FieldElement::into_mont` and then use it as input for `from_mont`

Reply:
thanks for this suggestion, understood the performance and potential errors related to string parsing during runtime if not done 🫡